graylog2-server/src/main/java/org/graylog2/configuration/ExposedConfiguration.java

```diff
@@ -88,6 +88,9 @@
     @JsonProperty("gc_warning_threshold")
     public abstract String gcWarningThreshold();
 
+    @JsonProperty("forwarder_enabled")
+    public abstract boolean forwarderEnabled();
```

Comment:
Do we actually need to expose that flag in the configuration API?
I think just having `isForwarderEnabled()` should be fine

Reply:
For the frontend to be able to decide whether to render/load or not related UI elements/code.

Reply:
Ah, makes sense :+1: